optimizely/event/processor.go

```diff
@@ -97,6 +97,10 @@ func (p *QueueingEventProcessor) StartTicker(ctx context.Context) {
 			case <-ctx.Done():
 				pLogger.Debug("Event processor stopped, flushing events.")
 				p.FlushEvents()
+				d, ok := p.EventDispatcher.(*QueueEventDispatcher)
```

Comment:
Hmm so on ticker we also try to flush the dispatcher? I think this coupling might be more than necessary. @pawels-optimizely @mikecdavis what do you all think? What about having the queue event dispatcher have its own ticker?

Reply:
no, on done from ctx we try and flush the dispatcher.  There is no timer or a need for one

Reply:
Oh yeah I misread. Makes sense.